app/src/main/java/com/example/eric/memberquiz/GameActivity.java

```diff
@@ -0,0 +1,306 @@
+package com.example.eric.memberquiz;
+
+
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.CountDownTimer;
+import android.provider.ContactsContract;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.w3c.dom.Text;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Random;
+
+public class GameActivity extends AppCompatActivity {
+
+    CountDownTimer timer, timer2;
+    Button button1;
+    Button button2;
+    Button button3;
+    Button button4;
+    Button endButton;
+    TextView score;
+    TextView countdownText;
+    SharedPreferences pref;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
Generally messy to put all code in onCreate method, use private methods and separate classes to divide up code